File: jdroid-android-core/src/main/java/com/jdroid/android/instagram/InstagramHelper.java
package com.jdroid.android.instagram;

import android.content.Intent;
import android.net.Uri;

import com.jdroid.android.activity.ActivityLauncher;

public class InstagramHelper {
	
	public static void openProfile(String account) {
		ActivityLauncher.startActivityNewTask(new Intent(Intent.ACTION_VIEW, Uri.parse("http://www.instagram.com/" + account)));
	}
	
}

File: jdroid-android-core/src/main/java/com/jdroid/android/instagram/InstagramHelper.kt
package com.jdroid.android.instagram

import android.content.Intent
import android.net.Uri

import com.jdroid.android.activity.ActivityLauncher

object InstagramHelper {

    fun openProfile(account: String) {
        ActivityLauncher.startActivityNewTask(Intent(Intent.ACTION_VIEW, Uri.parse("http://www.instagram.com/$account")))
    }
}
